parachain/test-parachains/undying/src/lib.rs

```diff
@@ -90,24 +90,42 @@ pub struct GraveyardState {
 	pub seal: [u8; 32],
 }
 
+/// A structure that configures HRMP messages produced by undying collator
+#[derive(Default, Clone, Encode, Decode, Debug)]
+pub struct HrmpChannelConfiguration {
+	/// Where to send HRMP messages
+	pub destination_para_id: u32,
+	/// Message size
```

Comment:
I imagine we can have a better interface here to configure the utilization of the channels. For example, can we also add number of messages ?

`HrmpChannel::{max_capacity, max_message_size, max_total_size}` already provide the maximum values, maybe we can use them

Reply:
I was there and there was such a parameter. However, there is a rule that there could be only one message per a combination of sender-recipient in a single candidate: https://github.com/paritytech/polkadot/blob/master/runtime/parachains/src/hrmp.rs#L941
So this parameter would be useless I suppose.

Reply:
This make sense and it would be nice to document this and show it to the user.